### PR TITLE
[C#] Correct error in coverage reporting

### PR DESCRIPTION
**Pull request by aman-goel**

### CODE REVIEW

**Review by ankushdesai — CHANGES_REQUESTED** [review]
Does this also fix the issue of in the coverage report, we dont print `PImplementation.X`

**Comment by aman-goel** [comment]
> Does this also fix the issue of in the coverage report, we dont print `PImplementation.X`

Done via [aef7289](https://github.com/p-org/P/pull/662/commits/aef7289387c2358720f498be05da8c7afcb7e0b2)

File: Src/PChecker/CheckerCore/Coverage/ActivityCoverageReporter.cs [hunk 1]
```diff
@@ -349,5 +353,15 @@ private static string GetMachineId(string nodeId)
 
         private static string GetStateId(string machineName, string stateName) =>
             string.Format("{0}::{1}", stateName, machineName);
+        
+        private static string GetSanitizedName(string name)
```
Comment (ankushdesai) [inline Src/PChecker/CheckerCore/Coverage/ActivityCoverageReporter.cs:357]: Instead of doing this can you do `name.split(".").Last()`

Comment (aman-goel) [reply Src/PChecker/CheckerCore/Coverage/ActivityCoverageReporter.cs:357]: Done via [d752c52](https://github.com/p-org/P/pull/662/commits/d752c5233a417fd23f38ac20bb8b074c907881e7)

**Pull request merged**